apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue

```diff
@@ -10,9 +10,11 @@
                         {{ $t('DASHBOARDS.WIDGET.TOTAL_FAILURE_AND_SEVERITY.TOTAL_FAILURE_COUNT') }}
                     </p>
                     <p class="value">
-                        {{ state.totalFailureCount }}
+                        {{ state.totalFailureCount ? state.totalFailureCount : '--' }}
```

Comment:
issue: Even the number `0` is `false` when automatically converted to boolean in JavaScript, so even if the value is `0` (not empty), it will be shown as ‘--’!